.github/actions/github-list-changes/github-list-changes.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -e
```

always add -e to make it resilient to random failures